client/Mac/CMakeLists.txt

```diff
@@ -134,15 +144,15 @@ if (${BUILD_SHARED_LIBS})
 			add_custom_command(TARGET ${MODULE_NAME} POST_BUILD
 				COMMAND install_name_tool -change "$<TARGET_SONAME_FILE:${LIB}>"
 					"@executable_path/../Frameworks/${MODULE_OUTPUT_NAME}.framework/Contents/$<TARGET_FILE_NAME:${LIB}>"
-					"${CMAKE_CURRENT_BINARY_DIR}/$(CONFIGURATION)/${MODULE_OUTPUT_NAME}.framework/Contents/$<TARGET_FILE_NAME:${DEST}>"
+					"${CMAKE_CURRENT_BINARY_DIR}/${safe_configuration}/${MODULE_OUTPUT_NAME}.framework/Contents/$<TARGET_FILE_NAME:${DEST}>"
 				COMMENT Setting install name for ${LIB} in module ${DEST})
 		endforeach()
 	endforeach()
 
 endif()
 
 # Add post-build NIB file generation in unix makefiles. XCode handles this implicitly.
-# if("${CMAKE_GENERATOR}" MATCHES "Unix Makefiles")
+# if(NOT "${CMAKE_GENERATOR}" MATCHES "Xcode")
```

Comment:
changes in a commented line?

Reply:
Yes. Adapt the code, if locally used or for whatever reasons, to apply to *all* generators that are not Xcode and not only to 'Unix Makefiles'